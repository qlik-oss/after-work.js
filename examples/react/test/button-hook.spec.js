import React from "react";
import renderer from "react-test-renderer";

const [{ default: ButtonHook }] = aw.mock([], ["../src/button-hook"]);

describe("Button with hook", () => {
  it("renders", () => {
    const tree1 = renderer.create(<ButtonHook>hook</ButtonHook>).toJSON();
    expect(tree1).toMatchSnapshot();
  });
});
