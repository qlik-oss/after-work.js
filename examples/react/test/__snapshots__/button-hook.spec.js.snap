// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`'renders button hook' 1`] = `
<span>
  hook
</span>
`;

exports[`'renders' 1`] = `
<div>
  <p>
    You clicked 
    0
     times
  </p>
  <button
    onClick={[Function]}
  >
    Click me
  </button>
</div>
`;
